packages/core/src/auto.ts

```diff
@@ -43,6 +43,7 @@ import loadPlugin, { IPlugin } from './utils/load-plugins';
 import createLog, { ILogger } from './utils/logger';
 import { makeHooks } from './utils/make-hooks';
 
+const proxyUrl = process.env.https_proxy;
```

Comment:
You mention in the docs that `http_proxy` or `https_proxy` both work, but you only use `https_proxy`. Should it be something more like:

```suggestion
const proxyUrl = process.env.https_proxy || process.env.http_proxy;
```

My question after that is: Does `HttpsProxyAgent` support `http_proxy`?

Reply:
Added support for http proxy as well